charts/renku/requirements.yaml

```diff
@@ -1,19 +1,19 @@
 dependencies:
 - name: renku-ui
   alias: ui
-  version: "0.2.0"
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
+  version: "0.2.0-592cc08"
 - name: renku-notebooks
   alias: notebooks
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.2.1-b0c93bb"
+  version: "0.2.1-3702f3b"
 - name: renku-gateway
   alias: gateway
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.2.0"
+  version: "0.2.0-9094770"
```

Comment:
can we do something about this not being `0.2.1-x`?

Reply:
sure, and the UI as well I suppose

Reply:
yup -> https://github.com/SwissDataScienceCenter/renku-ui/pull/344